spa/src/pages/ConfigSteps/index.tsx

```diff
@@ -307,7 +307,11 @@ const ConfigSteps = () => {
 																		onClick={() => {
 																			setLoaderForOrgClicked(true);
 																			setClickedOrg(org.id);
-																			doCreateConnection(org.id, "manual");
+																			try {
```

Comment:
Do we need the try catch here? `doCreateConnection` already has a try-catch. Can we just add the `finally` inside the `doCreateConnection`?

Reply:
?? It is the `finally` I added here.

Reply:
Although I might further refactor the button into sub componnent with loading states as well.